src/jdk.jartool/share/classes/sun/tools/jar/GNUStyleOptions.java

```diff
@@ -228,6 +228,20 @@ void process(Main jartool, String opt, String arg) {
                 }
                 boolean isHidden() { return true; }
             },
+            new Option(true, OptionType.OTHER, "--release") {
+                void process(Main jartool, String opt, String arg) throws BadArgs {
+                    try {
+                        int value = Integer.valueOf(arg);
+                        if (value < 9) {
+                            throw new BadArgs("error.release.value.toosmall", arg).showUsage(true);
+                        }
+                        jartool.releaseValue = value;
+                    } catch (NumberFormatException x) {
+                        throw new BadArgs("error.release.value.notnumber", arg);
+                    }
+                }
+                boolean isHidden() { return true; }
```

Comment:
Once we put this under `DESCRIBE_MODULE_OR_VALIDATE` `OptionType`, this option should not be hidden and should instead behave like a normal option for those main operations.